README.rst

```diff
@@ -82,29 +82,6 @@ Add your stripe keys:
     STRIPE_PUBLIC_KEY = os.environ.get("STRIPE_PUBLIC_KEY", "<your publishable key>")
     STRIPE_SECRET_KEY = os.environ.get("STRIPE_SECRET_KEY", "<your secret key>")
 
-Add some payment plans:
```

Comment:
You'll still need to document the process of installing plans via the database. It might be a good idea to keep the init_plans method to just take the plans set in settings.py and create model instances from them.